From b760c7214df9e772938a90f31a80fc3531b0d861 Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Wed, 10 Apr 2013 16:37:01 -0400
Subject: [PATCH] loosen pyparsing constraint

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c126dcd..b12665c 100644
--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@ install_requires = [
 # there is a release declare the dependency ourselves.
 # http://trac-hg.assembla.com/python-cmd2/rev/a5f3d5a89d6c
 if sys.version_info[0] < 3:
-    install_requires.append('pyparsing==1.5.7')
+    install_requires.append('pyparsing<=1.5.7')
 else:
     install_requires.append('pyparsing>=2.0.0')
 
-- 
1.8.1.4

